generator/pmdsky_debug_py_generator/loader.py

```diff
@@ -25,7 +25,8 @@
 import yaml
 
 OVERLAY_REGEX = re.compile(r"overlay(\d+)")
-C_TYPE_REGEX = re.compile(r"(((enum )|(struct ))?[a-z0-9_*]+) (([A-Z0-9_]+)(\[\d+])*);")
+C_TYPE_REGEX = re.compile(r"(extern\s+)?(?P<symbol_type>((enum)|(struct)\s+)?[a-z0-9_*]+)\s+"
```

Comment:
We ought to support the `const` keyword. There's already some instances of this with `extern const char*`

Reply:
Oh, I thought I added it already. I probably thought about it and forgot.
I'll add it to the regex. I'm thinking about putting it next to `extern`, outside of the `symbol_type` group, since `const` only applies to changing values at runtime, a ROM editor doesn't care about it.
Does that sound good?

Reply:
I would probably put optional optional const checks both before and after the symbol type, since `const` can go in either place, sometimes with different meanings. E.g.

```c
const char* non_const_ptr_to_const_char;
char* const const_ptr_to_non_const_char;
const char* const const_ptr_to_const_char;
```

Actually, `const` can even go in the middle, i.e. `char const*`, but it means the same thing as `const char*`, so it's probably okay to not support that.

Reply:
Alright, I've added it both before and after the type.